config/database.yml

```diff
@@ -1,16 +1,18 @@
-development:
+default: &default
   adapter: postgresql
-  database: growstuff_dev
-  host: localhost
+  encoding: unicode
+  host: db
```

Comment:
[Fully-native Docker for Windows/Mac](
https://blog.docker.com/2016/03/docker-for-mac-windows-beta/) is still in
beta, but Docker Toolbox appears to be non-beta on both [Windows](
https://docs.docker.com/engine/installation/windows/) and [OS X](
https://docs.docker.com/mac/step_one/). That runs Docker in a tiny Linux
VM, though, so developers on OS X and Windows would have to suffer
virtualization overhead. OTOH, our current suggestion for Windows is
"develop in a Linux VM".